src/vm.rs

```diff
@@ -311,28 +315,23 @@ impl<'a, E: UserDefinedError> EbpfVm<'a, E> {
     /// let mut vm = EbpfVm::<UserError>::new(executable.as_ref(), mem, &[]).unwrap();
     ///
     /// // Provide a reference to the packet data.
-    /// let res = vm.execute_program().unwrap();
+    /// let res = vm.execute_program_interpreted(&mut DefaultInstructionMeter {}).unwrap();
     /// assert_eq!(res, 0);
     /// ```
-    pub fn execute_program(&mut self) -> Result<u64, EbpfError<E>> {
-        self.execute_program_metered(DefaultInstructionMeter {})
-    }
-
-    /// Execute the program loaded, with the given instruction meter.
-    pub fn execute_program_metered<I: InstructionMeter>(
+    pub fn execute_program_interpreted<I: InstructionMeter>(
```

Comment:
Exposing the instructcion meter in the default case because the function name got too long?

Reply:
This way we only need two functions one for the interpreter and one for the JIT.
Otherwise, we would need four functions, because there would be a version with and without meter for each.